brigade-worker/test/k8s.ts

```diff
@@ -66,6 +66,26 @@ describe("k8s", function() {
     });
   });
 
+  describe("secretToProjectnoVCS", function() {
+    it("converts secret to project - without a VCS", function() {
+      let s = mockSecretnoVCS();
+      let p = k8s.secretToProject("default", s);
+      assert.equal(
+        p.id,
+        "brigade-0e0ae80bb1243d95ea707257baebda6ccf96094cd0353d875ae903"
+      );
+      assert.equal(p.name, "noVCSProject");
+      assert.equal(p.repo, undefined);
+      assert.equal(s.data["genericGatewaySecret"], "SThkQ1g=") // Project class does not contain genericGatewaySecret field - this is because we do not want to expose the genericGatewaySecret to any job
```

Comment:
At first I tried to asset `p.genericGatewaySecret` but I discovered that this property is not exposed in Project class in Brigadier ([link](https://github.com/brigadecore/brigadier/blob/master/src/events.ts#L199))
I debated whether to PR Brigadier and add it, but ultimately I decided against it. Reasoning is that I can't find any legit reason to expose `genericGatewaySecret` to a job running within a brigade.js file. WDYT?